modules/ROOT/pages/amazon/amazon-dynamodb-connector.adoc

```diff
@@ -8,298 +8,64 @@ Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#
-xmlns="http://www.mulesoft.org/schema/mule/core"
-xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
-xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core
-http://www.mulesoft.org/schema/mule/core/current/mule.xsd
-http://www.mulesoft.org/schema/mule/dynamodb
-http://www.mulesoft.org/schema/mule/dynamodb/current/mule-dynamodb.xsd
-http://www.mulesoft.org/schema/mule/http
-http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd">
-    <configuration-properties file="mule-artifact.properties" />
-	<dynamodb:config 
-		name="Amazon_DynamoDB_Configuration" 
-		doc:name="Amazon DynamoDB Configuration">
-		<dynamodb:basic-connection 
-			accessKey="${config.accesskey}" 
-			secretKey="${config.secretkey}" />
-	</dynamodb:config>
-	<http:listener-config 
-		name="HTTP_Listener_config" 
-		doc:name="HTTP Listener config">
-		<http:listener-connection host="localhost" 
-		port="8091" />
-	</http:listener-config>
-	<flow name="create-table-flow">
-		<http:listener 
-			doc:name="Listener" 
-			config-ref="HTTP_Listener_config" 
-			path="/createtable"/>
-		<dynamodb:create-table doc:name="Create table"
-		config-ref="Amazon_DynamoDB_Configuration"
-		tableName="StudentData"
-		readCapacityUnits="5" writeCapacityUnits="5">
-			<dynamodb:attribute-definitions >
-				<dynamodb:attribute-definition 
-					attributeName="studentId" 
-					attributeType="N" />
-			</dynamodb:attribute-definitions>
-			<dynamodb:key-schemas >
-				<dynamodb:key-schema-element 
-					attributeName="studentId" 
-					keyType="HASH" />
-			</dynamodb:key-schemas>
-		</dynamodb:create-table>
-		<logger level="INFO" doc:name="Logger" message="#[payload]"/>
-	</flow>
-	<flow name="delete-table-flow">
-		<http:listener 
-			doc:name="Listener" 
-			config-ref="HTTP_Listener_config" 
-			path="/delete"/>
-		<dynamodb:delete-table doc:name="Delete table"
-		config-ref="Amazon_DynamoDB_Configuration" tableName="StudentData"/>
-		<logger level="INFO" doc:name="Logger" message="#[payload]"/>
-	</flow>
-</mule>
-----
+== Authentication Types
 
-== Migration information
+Amazon DynamoDB Connecor supports the Basic and Role authentication types. For information about configuring these authentication types, see xref:amazon/amazon-dynamodb-connector-studio.adoc[Use Studio to Configure Amazon DynamoDB Connector].
 
-Changes from version *1.3.0* to *1.4.0* include the following:
+== Using Exchange Templates and Examples
 
-- Added TLS configuration into the connector.
-- Upgraded `aws-java-sdk` from 1.11.604 to 1.11.649
-- Upgraded `aws-connector-commons` from 2.0.2 to 2.2.0
+https://www.mulesoft.com/exchange/[Anypoint Exchange] provides templates
+you can use as a starting point for your app, as well as examples that illustrate a complete solution.
 
+== Next Step
 
+After you complete the prerequisites and experimenting with templates and examples, you are ready to create a new app using Anypoint Studio.
```

Comment:
experimenting or experiment?

Reply:
I will add "finish" as in "finish experimenting."